actions/global_actions.jsx

```diff
@@ -239,6 +240,8 @@ export function emitUserLoggedOutEvent(redirectTo = '/', shouldSignalLogout = tr
         LocalStorageStore.setWasLoggedIn(false);
     }
 
+    dispatch(setPreviousRecentEmojis());
```

Comment:
Placed it here instead of after successful logout so that the currentUserId is still available on store.